src/dx/config.h

```diff
@@ -1,6 +1,9 @@
 #ifndef DX
 #define DX
 
+/// Set the version string that appears on the file select menu.
+#define DX_MOD_VERSION_STRING "??? ?.?.?"
```

Comment:
Turn this off by default, and give a sensible example. Something like
```suggestion
/// Version string that appears on the file select menu.
//#define DX_MOD_VERSION_STRING "Awesome romhack 0.1.0"
```

Reply:
Added a default name "New Mod" and an explanation that you can comment it out to disable it. I think modders should be encouraged to display this information by default, though obviously not be required to do so.